src/Core/src/Converters/SafeAreaEdgesTypeConverter.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.ComponentModel;
+using System.Globalization;
+
+namespace Microsoft.Maui.Converters
+{
+	/// <summary>
+	/// Type converter for SafeAreaEdges struct, supporting XAML parsing of 
+	/// SafeArea.Ignore attached property values.
+	/// </summary>
+	public class SafeAreaEdgesTypeConverter : TypeConverter
+	{
+		public override bool CanConvertFrom(ITypeDescriptorContext? context, Type sourceType)
+			=> sourceType == typeof(string);
+
+		public override bool CanConvertTo(ITypeDescriptorContext? context, Type? destinationType)
+			=> destinationType == typeof(string);
+
+		public override object ConvertFrom(ITypeDescriptorContext? context, CultureInfo? culture, object? value)
+		{
+			// IMPORTANT! Update SafeAreaEdgesTypeDesignConverter.IsValid if making changes here
+			var strValue = value?.ToString();
+
+			if (strValue != null)
+			{
+				strValue = strValue.Trim();
+				
+				// Split by comma - if no comma, we get array with single element
+				var parts = strValue.Split(',');
+				var regions = new SafeAreaRegions[parts.Length];
+
+				for (int i = 0; i < parts.Length; i++)
+				{
+					var part = parts[i].Trim();
+					
+					// Performance optimization: use string comparison instead of Enum.TryParse
```

Comment:
I thought Enum.TryParse no longer had pref issues... and we use it elsewhere. A real perf bump would be to provide a compiled version of this typeconverter, as it will greatly reduce the number of allocation, the the byte size to JIT, and the number of instructions to execute at runtime